### PR TITLE
refactor(framework) Install app from `.whl` in `FAB`

### PR DESCRIPTION
**Pull request by jafermarq**
Merge after https://github.com/adap/flower/pull/4475

### CODE REVIEW

File: src/py/flwr/cli/install.py [hunk 1]
```diff
@@ -188,23 +188,25 @@ def validate_and_install(
...
-            ["pip", "install", "-e", install_dir, "--no-deps"],
+            ["pip", "install", "--no-deps", install_whl],
             capture_output=True,
             text=True,
             check=True,
         )
     except subprocess.CalledProcessError as e:
         typer.secho(
-            f"❌ Failed to `pip install` package(s) from {install_dir}:\n{e.stderr}",
+            f"❌ Failed to `pip install` whell {install_whl}:\n{e.stderr}",
```
Comment (chongshenng) [inline src/py/flwr/cli/install.py:202]:
```suggestion
            f"❌ Failed to `pip install` wheel {install_whl}:\n{e.stderr}",
```

**Review by chongshenng — COMMENTED** [review]
Small comment, otherwise it LGTM.

File: src/py/flwr/cli/install.py [hunk 2]
```diff
@@ -188,23 +188,25 @@ def validate_and_install(
...
-            f"❌ Failed to `pip install` package(s) from {install_dir}:\n{e.stderr}",
+            f"❌ Failed to `pip install` wheel {install_whl}:\n{e.stderr}",
             fg=typer.colors.RED,
             bold=True,
         )
         raise typer.Exit(code=1) from e
 
     typer.secho(
-        f"🎊 Successfully installed {project_name} to {install_dir}.",
+        f"🎊 Successfully installed {project_name} from {install_whl}.",
```
Comment (danieljanes) [inline src/py/flwr/cli/install.py:209]:
```suggestion
        f"🎊 Successfully installed {project_name}.",
```

File: src/py/flwr/cli/install.py [hunk 3]
```diff
@@ -188,23 +188,25 @@ def validate_and_install(
...
-            ["pip", "install", "-e", install_dir, "--no-deps"],
+            ["pip", "install", "--no-deps", install_whl],
             capture_output=True,
             text=True,
             check=True,
         )
     except subprocess.CalledProcessError as e:
         typer.secho(
-            f"❌ Failed to `pip install` package(s) from {install_dir}:\n{e.stderr}",
+            f"❌ Failed to `pip install` wheel {install_whl}:\n{e.stderr}",
```
Comment (danieljanes) [inline src/py/flwr/cli/install.py:202]:
```suggestion
            f"❌ Failed to install {project_name}:\n{e.stderr}",
```

**Pull request merged**